Modules/ASEP/Get-SchedTasks.ps1

```diff
@@ -3,7 +3,11 @@
 Get-ShedTasks.ps1 returns information about all Windows Scheduled Tasks.
 .NOTES
 The following line is required by Kansa.ps1, which uses it to determine
-how to handle the output from this script.
-OUTPUT tsv
```

Comment:
This should stay in. It's a directive the Kansa engine uses to determine output format. Modules should always return a PowerShell object.